packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -282,13 +298,28 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     return null;
   }
 
+  // deprecated
   private _onRenderAddon(props: ITextFieldProps): JSX.Element {
```

Comment:
don't have duplicate code :) Just point both at the same private callback.

Reply:
I did this because onRenderAddon still needs to use the addonString property. I'd need to add some custom logic into onRenderPrefix to decide which property to use. Would that be preferred?